project/src/components/place-card/place-card.jsx

```diff
@@ -1,48 +1,69 @@
 import React from 'react';
+import PropTypes from 'prop-types';
+import PremiumLabel from '../premium-label/premium-label';
+import CardPhoto from './card-photo/card-photo';
+import {getCardsInfo} from './getCardsInfo';
+import {cardInfoProp} from '../../types/cards.prop';
+
+const classNames = require('classnames');
+
+const bigImage = {
+  width: 260,
+  height: 200,
+};
```

Comment:
```suggestion
const BigImage = {
  WIDTH: 260,
  HEIGHT: 200,
};
```
по договоренности следует использовать наглядную ~агитацию~ капитализациюь и уносить такие определения в отдельные модули

Reply:
Максим, вот с этими стилями никак не могу определиться. Должна быть концепция, которой я бы придерживалась при создании компонентов. Сейчас вижу две концепции. 
1. Кому как не Родителю, при использовании компонента знать, какие стили задавать. Поэтому, передавать стили (проблема же в том, что их несколько), в виде объекта компоненту-children. Тогда компонент просто берет их и использует. Нет проблемы выбора и подстановки стилей. Но Вы сказали, что лучшая практика, это когда простые типы передаются компонентам.
2. Передавать флаг стиля. Тогда компонент-chldren по этому флагу выбирает нужные стили. А все используемые стили хранит у себя. 
В общем, прихожу к тому, чтобы создать файл со стилями в папке компонента и грузить нужный оттуда.

Reply:
это в тебе говорит профессионал! В общем правильно, нужно что-то предоставить "наружу". Лучше показывать меньше.